setup.cfg

```diff
@@ -67,7 +67,7 @@ filterwarnings =
     error
     # Suppress deprecation warnings in other packages
     ignore:lib2to3 package is deprecated::scspell
-    ignore:pkg_resources is deprecated as an API::pkg_resources
+    ignore:pkg_resources is deprecated as an API
```

Comment:
This will get dropped in https://github.com/colcon/colcon-core/pull/562/files#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52L70 so I'm not worried that we go from scoped to completely unscoped here.

Reply:
Sounds good. The goal is simply to get green CI to test the other related changes prior to dropping it entirely.